AirCasting/AirCastingApp.swift

```diff
@@ -39,11 +41,15 @@ struct AirCastingApp: App {
         }.onChange(of: scenePhase) { newScenePhase in
             switch newScenePhase {
             case .active:
-                shouldProtect = false
+                if mobilePeripheralSessionManager.isMobileSessionActive || microphoneManager.isRecording {
```

Comment:
I think this check is not necessary? And maybe it is safer to not include it, because for example if the session stops being active when the app is in the background, then shouldProtect won't be set to false when the app comes back to foreground and we would want that to happen

Reply:
Nice point of looking at that. I will delete this condition from here